app/Gists/File.php

```diff
@@ -0,0 +1,64 @@
+<?php namespace Gistlog\Gists;
+
+use Illuminate\Contracts\Support\Arrayable;
+
+class File implements Arrayable
+{
+    public $name;
+    public $type;
+    public $language;
+    public $url;
+    public $size;
+    public $content;
+
+    /**
+     * @param array|ArrayAccess $rawFile
+     * @return File
+     */
+    public static function fromGitHub($rawFile)
+    {
+        $file = new self;
+
+        $file->name = $rawFile['filename'];
+        $file->type = $rawFile['type'];
+        $file->language = $rawFile['language'];
+        $file->url = $rawFile['raw_url'];
+        $file->size = $rawFile['size'];
+        $file->content = $rawFile['content'];
+
+        return $file;
+    }
+
+    /**
+     * @param array|ArrayAccess $rawFiles
+     * @return Collection
+     */
+    public static function multipleFromGitHub($rawFiles)
+    {
+        // Remove the blog post itself.
+        array_shift($rawFiles);
```

Comment:
@mattstauffer The only post in which you recommend using `blog.md` is in this one here [Introducing Gistlog Landing Pages](https://gistlog.co/mattstauffer/a82db51468a02f53442d) which is only accessible by going to your own landing page [http://gistlog.co/MattStauffer](http://gistlog.co/MattStauffer) and clicking on the link.

That being said, I don't think it is overly obvious to a newcomer that they need to name their markdown file `blog.md` although we could certainly encourage and likely enforce that through better documentation on how to use the platform. I would like to do this when I get around to re-doing the homepage and eventually add some documentation. 

Looking at [this fixture](https://github.com/dannyweeks/gistlog/blob/feature/optionally-include-additional-files/tests/fixtures/gists/aac5edd61c183dd26392.json) though, it does seem that the files are listed alphabetically. Don't think we can currently rely on users having `blog.md`, even if that is the name they are using.

I think the best path forward is to define the opinions and opinions of Gistlog up front. Something as simple as [http://bl.ocks.org/-/about](http://bl.ocks.org/-/about).